package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
```

Comment:
I believe You should only have changed the ./src file. I might be wrong tho. Would be safe to remove changes made in the package.json and package-lock.json files before the mentor review. :D